src/net/util/show_packet.h

```diff
@@ -20,7 +20,7 @@ static inline void show_packet(uint8_t *raw, int size, char *title) {
 	if (!&mod_logger)
 		return;
 
-	if (&mod_logger.logging &&
+	if (//&mod_logger.logging &&
```

Comment:
Checking for `&whatever` to be non-null is indeed redundant, I guess, this should be removed. AFAICS, this file, and these lines, in particular, were added by @Deryugin, Denis, could you elaborate on this, please?